src/maud/stan_code/allostery.stan

```diff
@@ -77,4 +77,6 @@ real get_free_enzyme_ratio_ordered_terbi(real A, real B, real C, real P, real Q,
     + Ka*V2*B*C*P*Q/(Kip*Kiq);
   return num / denom;
 }
-
+real get_free_enzyme_ratio_modular_rate_law(real Tr, real Dr){
```

Comment:
It's to simplify the way free enzyme ratio functions are currently called. The catalytic string is used for the arguments and it would be more of an effort to include an extra clause just for the modular rate law.